index.js

```diff
@@ -8,6 +8,15 @@ module.exports = {
   improve: 'apostrophe-rich-text-widgets',
 
   beforeConstruct: function(self, options) {
+    options.sanitizeHtml = options.sanitizeHtml || {};
+
+    options.sanitizeHtml.allowedAttributes = {
+      ...options.sanitizeHtml.allowedAttributes,
+      a: ['href', 'name', 'target'],
```

Comment:
This will throw out any user-configured attributes for the "a" tag.

If you wish to implement a merge of this kind you will need to use something like lodash merge.

But I think it would be better to stick to 2.x practice for regular rich text widgets: if you touch `allowedAttributes` at all, you must specify everything. Or you could explicitly add new `addAllowedAttributes`, etc. options, and use `_.merge` with those.